src/compiler/transformers/ts.ts

```diff
@@ -1194,6 +1194,11 @@ namespace ts {
             );
         }
 
+        function isPrivateProperty(member: ClassElement): member is PropertyDeclaration {

```

Comment:
The return type of the function doesn't match what it does: the body checks not just that the member is a property declaration, but that the members' name is a private name. So which is right, the return value or the behavior?

Also, the name could be confusing because `private` properties are also private properties: maybe name it something like "privateNamedProperty" would be better?